jinahub/segmenters/VADSpeechSegmenter/vad_speech_segmenter.py

```diff
@@ -0,0 +1,135 @@
+__copyright__ = 'Copyright (c) 2020-2021 Jina AI Limited. All rights reserved.'
+__license__ = 'Apache-2.0'
+
+from pathlib import Path
+from typing import Optional, Dict
+
+import torch
+import torchaudio
+import numpy as np
+import librosa as lr
+from jina import Executor, Document, DocumentArray, requests
+from jina.excepts import BadDocType
+
+
+class VADSpeechSegmenter(Executor):
+    """
+     Segment the speech audio using Silero's Voice Activity Detector (VAD)
+
+    :param normalize: a bool to specify whether to normalize the audio
+     by the sample rate
+    :param dump: a bool to specify whether to dump the segmented audio
+    :param repo: default repo name of silero-vad
+    :param model: default model name of silero-vad
+
+    """
+
+    TARGET_SAMPLE_RATE = 16000
+
+    def __init__(
+        self,
+        normalize: bool = False,
+        dump: bool = False,
+        repo: str = 'snakers4/silero-vad',
+        model: str = 'silero_vad',
+        *args,
+        **kwargs,
+    ):
+        super().__init__(*args, **kwargs)
+        if model is None or repo is None:
+            raise ValueError('model and repo cannot be None')
+
+        # TODO: remove the following temporary fix for torch hub
+        torch.hub._validate_not_a_forked_repo = lambda *args: True
+        self.model, utils = torch.hub.load(
+            repo_or_dir=repo, model=model, force_reload=True
+        )
+
```

Comment:
```suggestion
self.model.eval()
```
Please make sure to perform the inference in `eval` mode

Reply:
Addressed, please see [here](https://github.com/jina-ai/executors/pull/97/commits/752f958d7e44f07d635699131e1a7a43febce5ab) for the change.